### ISSUE TITLE
Autosign crashes game

### ISSUE THREAD

**Issue by TheWienerMaster** [Wurst-Imperium/Wurst7#429]
## Steps To Reproduce
1. Enable Autosign
2. Create the first sign
3. Place down the second sign
4. Crash

## Crash Report

```
java.lang.IllegalAccessError: Update to non-static final field net.minecraft.class_498.field_24285 attempted from a different method (handler$zgg004$onInit) than the initializer method <init> 
	at net.minecraft.class_498.handler$zgg004$onInit(class_498.java:544)
	at net.minecraft.class_498.method_25426(class_498.java)
	at net.minecraft.class_437.method_25423(class_437.java:387)
	at net.minecraft.class_310.method_1507(class_310.java:986)
	at net.minecraft.class_746.method_7311(class_746.java:585)
	at net.minecraft.class_634.method_11108(class_634.java:1049)
	at net.minecraft.class_2693.method_11676(class_2693.java:25)
	at net.minecraft.class_2693.method_11054(class_2693.java:7)
	at net.minecraft.class_2600.method_11072(class_2600.java:21)
	at net.minecraft.class_1255.method_18859(class_1255.java:151)
	at net.minecraft.class_4093.method_18859(class_4093.java:23)
	at net.minecraft.class_1255.method_16075(class_1255.java:125)
	at net.minecraft.class_1255.method_5383(class_1255.java:110)
	at net.minecraft.class_310.method_1523(class_310.java:1077)
	at net.minecraft.class_310.method_1514(class_310.java:730)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)

```

**Comment by TheWienerMaster** [comment]
The problem seems to originate from onFinishEditing from EditSignScreenMixin, since disabling WurstClient.INSTANCE.getHax().autoSignHack.setSignText(text); doesn't set the text anymore, but also doesn't crash the game anymore.

**Comment by Alexander01998** [comment]
This is fixed as of [Wurst 7.20](https://www.wurstclient.net/updates/wurst-7-20/).
